Network/Network.cpp

```diff
@@ -486,15 +486,15 @@ namespace WPEFramework
                     string interface = "";
                     bool persist = false;
 
-                    getStringParameter("interface", interface)
+                    getDefaultStringParameter("interface", interface, "")
```

Comment:
This is already being done outside. I understand it's good to do it within this getDefaultStringParameter API but checking how does this help with the crash.